applications/metaquity_nft_provenance_tracker.md

```diff
@@ -0,0 +1,148 @@
+
+Later on we can extend EVM compatibility to this project to make it more versatile.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Abhishek Chatterjee
+- Abhishek Sira Chandrashekhar
+- Arko Ganguli
+
+### Contact
+
+- **Contact Name:** Abhishek Chatterjee
+- **Contact Email:** abhishek@metaquity.xyz
+- **Website:** [Metaquity](www.metaquity.xyz/)
+
+### Legal Structure
+
+- **Registered Address:** 128 CITY ROAD, UNITED KINGDOM, LONDON, EC1V 2NX
+- **Registered Legal Entity:** Metaquity Limited.
+
+### Team's experience
+
+- **Abhishek Chatterjee**: Co-founder of Metaquity, 15+ years of product and technical experience. 4+ years of blockchain experience. 
+- **Abhishek Sira Chandrasekhar**: Software developer with 4+ years of experience in development 2+ years in Blockchain working on DID, SSI, VC and ZKP. Prior to Metaquity used to work at Litentry as a mobile developer.
+
+- Metaquity Network is already part of the Substrate Builders Program.
+
+
+### Team Code Repos
+
+- https://github.com/metaquiry-network/nft-provenance-tracker
+
+Team members GitHub:
+
+- https://github.com/abhichat85
+- https://github.com/ekaanth
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/abhishekchatterjee85/
+- https://www.linkedin.com/in/abhishek-sira-chandrashekar-821244b7/
+
+## Development Roadmap :nut_and_bolt:
+
+This project will be a proof of concept (PoC) to demonstrate a particular use case and how NFT-SDK can be leveraged by other projects and parachains in the Polkadot ecosystem. Since this will be an SDK, anyone will be able to integrate this within their application. This acts as an enabler for this particular use case for provenance tracking of NFTs and fractional fungible tokens associated with those NFTs. This can extend to not only digital assets but also real-world assets (which is what is the primary focus of Metaquity Network).
+
+### Overview
+
+- **Total Estimated Duration:** 2,5 months
+- **Full-Time Equivalent (FTE):**  2 FTE (full-time employees)
+- **Total Costs:** 20,000 USD
+
+### Milestone 1 
+
+- **Estimated duration:** 1 month
+- **FTE:**  1,5
+- **Costs:** 10,000 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both readme.md and official docs documentation |
+| **0c.** | Testing and Testing Guide | Testing guide will be mentioned in official docs & core unit tests will be provided |
+| 1. | Data model definition | Designing and defining the architectural overview of the data model for this application  |
+| 2. | NFT scanner module | This module will be in middle layer to scan the blockchain for the blocks containing NFT related data from the genesis block and invoke indexer module (We will be indexing only blocks that has NFT related data.)|
```

Comment:
The previous, related [inline comment](https://github.com/w3f/Grants-Program/pull/1832#discussion_r1294560239) still hasn't been addressed.